cpp/example_code/cross-service/README.md

```diff
@@ -0,0 +1,22 @@
+# AWS SDK for C++ cross-service examples 
+
+## Overview
+This README lists the cross-service examples available for the AWS SDK for C++. Each folder in this directory contains the following cross-service examples. A README in each folder describes how to run the example.
+
+A cross-service example is an application that works across multiple AWS services using the AWS SDK for C++.
+
+
+## ⚠️ Important
+* Running this code might result in charges to your AWS account. 
+* Running the tests might result in charges to your AWS account.
+*  We recommend that you grant your code least privilege. At most, grant only the minimum permissions required to perform the task. For more information, see [Grant least privilege](https://docs.aws.amazon.com/IAM/latest/UserGuide/best-practices.html#grant-least-privilege). 
+* This code is not tested in every AWS Region. For more information, see [AWS Regional Services](https://aws.amazon.com/about-aws/global-infrastructure/regional-product-services).
+
+## Cross-service examples
+* [Amazon Aurora Severless item tracker](serverless-aurora/README.md) Create an item tracker web app using Amazon Relational Database Service (Amazon RDS), and Amazon Simple Email Service (Amazon SES).
```

Comment:
Amazon Relational Database Service (Amazon RDS) -> Amazon Aurora Serverless
I suppose you might also mention use of Amazon Relational Database Service (Amazon RDS) Data Service to run queries against the database.